package.json

```diff
@@ -58,7 +58,8 @@
   "scripts": {
     "commit": "git-cz",
     "build": "tsc && copyfiles \"lib/**/*.json\" dist && copyfiles \"tests/**/*.js*\" dist && copyfiles \"package.json\" dist",
-    "test": "npm run build && nyc jasmine --config=./tests/jasmine.json",
+    "test": "npm run build && jasmine --config=./tests/jasmine.json",
+    "test:coverage": "npm run build && nyc jasmine --config=./tests/jasmine.json",
```

Comment:
Pushing back on this change. Why postpone test coverage report when you can get it in the same run with practically no impact on execution time? (I had the same way as yours, and was convinced in my own PR by another reviewer that having code coverage in the same report is the way to go)

Reply:
when focusing tests, why look at a coverage report that is 99% irrelevant?

Reply:
Just don't look at it? This is more for the case that people forget to run coverage and as a result coverage decreases over time. I am not gonna block this PR over this little thing, so I'll let you make the call, but my 2 cents here is that unless you are going to touch this codebase and run these tests frequently from now on, I think it makes sense to let the current practice be the default. Maybe a good compromise would be to introduce `test:no-cc` instead.

Reply:
you can set the PR to fail if coverage drops below the required threshold, and then apply that to every PR, so that nobody can choose to ignore coverage :)

But I will also happy to revert this change, to `test:only`, so thats its easy quickly run tests without seeing coverage that is going to be 0 since the tests are focused.